### DISCUSSION TITLE
What's the correct way to migrate brew from one profile to another on macOS?

### DISCUSSION THREAD

**Discussion by hoseNdoW** [Homebrew/discussions#5639] [Getting started]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.3.24
ORIGIN: https://github.com/Homebrew/brew
HEAD: 916044581862c32fc2365e8e9ff0b1507a98925e
Last commit: 5 days ago
Core tap JSON: 27 Sep 16:15 UTC
Core cask tap JSON: 27 Sep 16:15 UTC
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_EDITOR: nvim
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 3.3.4 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/3.3.4_1/bin/ruby
CPU: octa-core 64-bit icelake
Clang: 16.0.0 build 1600
Git: 2.46.2 => /usr/local/bin/git
Curl: 8.7.1 => /usr/bin/curl
macOS: 15.0-x86_64
CLT: 16.0.0.0.1.1724870825
Xcode: N/A
```

### Output of `brew doctor`

```shell
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Some installed formulae are deprecated or disabled.
You should find replacements for the following formulae:
  openssl@1.1

Warning: You have unlinked kegs in your Cellar.
Leaving kegs unlinked can lead to build-trouble and cause formulae that depend on
those kegs to fail to run properly once built. Run `brew link` on these:
  tree-sitter

Warning: Homebrew's "sbin" was not found in your PATH but you have installed
formulae that put executables in /usr/local/sbin.
Consider setting your PATH for example like so:
  echo 'export PATH="/usr/local/sbin:$PATH"' >> ~/.zshrc
```

### Description of issue

On my Mac I have 2 accounts, both with admin rights. At this moment, the account for which `brew` was installed is abandoned, and I plan to delete it, leaving only one account. Currently, to install something, I use the command `su [abandoned_user] -c 'brew [command]'`.

The problem is that, as far as I know, if you delete `brew`, it will delete all installed applications along with it, as well as all their settings (I'm not 100% sure about this).

Question: What's the best course of action for me in this situation? Should I change the owner for everything related to `brew` using `chown`, or should I delete `brew` and reinstall it for the second account?

**Comment by carlocab** [comment]
You should just be able to `chown` it. If you run `brew doctor` from the new account (without `su`), it'll tell you the exact `chown` command you should run.
